src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -52,15 +61,99 @@ After a customer puts a pair of shoes in their cart and decides it's time to che
 
 To figure out why customers can't click the `Pay Now` button, open APM and select your `Checkout-service`. 
 
-
 <Steps>
 <Step>
+## Identify painpoints
+Before you dive into the specifics, it's important to first look at the summary tiles at the top of the page. These tiles alert you to any issues, vulnerabilities, or deployment failures in your system immediately. If the tiles on the top of your page show an empty state, click `Set up now` to activate each tile. 
+
+<img
+    title="apm summary tiles"
+    alt="A screenshot depicting the apm summary tiles"
+    src={apmApmSummaryTiles}
+/>
+
+<CollapserGroup>
+  <Collapser
+    id="issues"
+    title="Issues"
+  >
+
+<img
+    title="issues summary tile "
+    alt="A screenshot depicting the issues summary tile"
+    src={apmIssuesSummaryTile}
+/>
+
+An important way to track unusual behavior in your system is through alerts. Let's say  you want to know when the transaction error rate for `Checkout-service` exceeds 10%. You'd create an [alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/) and set the rules that will trigger an incident. One or more incidents creates an issue. This tile will display any issues in your service.
+
+In this example, you've already set up your alert conditions so when you scan your summary tiles you immediately see that your system has 2 critical issues. Click the tile for details.
+
+</Collapser>
+
+<Collapser
+    id="deployments"
+    title="Last deployment"
+  >
+
+<img
+    title="deployment summary tile"
+    alt="A screenshot showing the deployment summary tile"
+    src={apmDeploymentSummaryTile}
+/>
+Deployment markers help you monitor the results of every update you make to your service. This tile shows the change in error rate and response time triggered by your last deployment.
+
+Starting 15 minutes after a deployment, this tile will display a comparison between data collected before the deployment and after. For the first 3 hours after a deployment, this tile will only display a comparison of time of passed since enablement matched with the corresponding time before enablement. For example, if you enabled a new feature 45 minutes ago, this tile will show a comparison between the 45 minutes before and 45 minutes after. After three hours, the tile uses the standard 3-hour comparison.
```

Comment:
> For the first 3 hours after a deployment, this tile will only display a comparison of time of passed since enablement matched with the corresponding time before enablement. 

This one is minor, but I'd rework this sentence a bit. It makes sense after I read the explanation that follows it, but I reread this like five times because I didn't understand only for it to make sense instantly when I read the next sentence.

Reply:
Lol yes, @bradleycamacho this took forever to work out and it looks like I still didn't nail it!